Client/Frontend/Menu/BookmarksViewController.swift

```diff
@@ -566,10 +566,12 @@ extension BookmarksViewController: NSFetchedResultsControllerDelegate {
   func controller(_ controller: NSFetchedResultsController<NSFetchRequestResult>, didChange anObject: Any, at indexPath: IndexPath?, for type: NSFetchedResultsChangeType, newIndexPath: IndexPath?) {
     switch type {
     case .update:
-      guard let indexPath = indexPath, let cell = tableView.cellForRow(at: indexPath) else {
-        return
-      }
-      configureCell(cell, atIndexPath: indexPath)
+        if let indexPath = indexPath, let cell = tableView.cellForRow(at: indexPath) {
```

Comment:
I could have sworn I left a review for this. Ugh.

wdyt about:

```
let config = { (path: IndexPath?) in
    guard let path = path, let cell = self.tableView.cellForRow(at: path) else { return }
    self.configureCell(cell, atIndexPath: path)
}
[indexPath, newIndexPath].forEach(config)
```

I hate duplicate code 😄

Reply:
yeah you did in another FRC delegate :D, I updated the code